Assets/MixedRealityToolkit.Services/DiagnosticsSystem/MixedRealityToolkitVisualProfiler.cs

```diff
@@ -0,0 +1,485 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Diagnostics;
+using System.Text;
+using UnityEngine;
+using UnityEngine.Profiling;
+
+namespace Microsoft.MixedReality.Toolkit.Services.DiagnosticsSystem
+{
+    /// <summary>
+    /// ABOUT: The VisualProfiler provides a drop in, single file, solution for viewing 
```

Comment:
I verified this can still be dropped into a scene and works standalone. 👍

Reply:
Thanks! You are awesome, @Cameron-Micka!